packages/app-store/routing-forms/config.json

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
Removing routing forms from app store config since it's not part of app store anymore